llap-server/src/java/org/apache/hadoop/hive/llap/cache/LowLevelCachePolicy.java

```diff
@@ -26,6 +28,32 @@
  */
 public interface LowLevelCachePolicy extends LlapIoDebugDump {
 
+  static LowLevelCachePolicy provideFromConf(Configuration conf) {
+    final long totalMemorySize = HiveConf.getSizeVar(conf, HiveConf.ConfVars.LLAP_IO_MEMORY_MAX_SIZE);
+    final int minAllocSize = (int) HiveConf.getSizeVar(conf, HiveConf.ConfVars.LLAP_ALLOCATOR_MIN_ALLOC);
+    String policyName = HiveConf.getVar(conf,HiveConf.ConfVars.LLAP_IO_CACHE_STRATEGY);
```

Comment:
would it make sense to lowerCase the conf?

Reply:
you mean the policyName to policyname ?

Reply:
I meant the value, so that FiFo is the same as fifo